### PR TITLE
#181339467 Set up an empty Express Boilerplate with dotenv

### PR DESCRIPTION
**Pull request by Gabin-ishimwe**
### What does this PR do?
Create a boiler plate of our project setup
Added babel to compile and use es6 javascript codes
### Description of Task to be completed?
merge this boiler plate PR so that other team member can start working on it
### How should this be manually tested?

- Clone this repository and open the project in your code editor.
- to install all package on your local repo use: **yarn install** 
- to run the project use: **yarn run dev**

### Any background context you want to provide?
after the PR is merged, clone it and create your own dotenv(.env) file to manage environments variables needed in the project.

I have added a babel plugin called module resolver which will help to make absolute import instead of declaring the path  of the file when importing (check screenshot below)

### What are the relevant pivotal tracker stories?
[#181339467](https://www.pivotaltracker.com/n/projects/2555280)

### CODE REVIEW

**Comment by Eli250** [comment]
@Gabin-ishimwe, make sure that you have added a description on how to manually test this PR.

**Review by Eli250 — APPROVED** [review]

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    "presets": ["@babel/preset-env"],
+    "plugins": [
+        [
+            "module-resolver",
+            {
+                "root": ["./src"]
+            }
+        ]
+    ]
+}
```
Comment (Code-250) [inline .babelrc]: add new line at the end

**Review by Code-250 — APPROVED** [review]

**Comment by ncutixavier** [comment]
### Comment
- Your PR has conflicts
- Describe well how to run your code locally
-

**Comment by Gabin-ishimwe** [comment]
> ### Comment
> * Your PR has conflicts
> * Describe well how to run your code locally

#### conflict resolved
#### described how to run the project

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**